src/main/kotlin/app/batch/MessageUtils.kt

```diff
@@ -0,0 +1,74 @@
+package app.batch
+
+import com.beust.klaxon.JsonObject
+import com.beust.klaxon.Parser
+import com.beust.klaxon.lookup
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Component
+import java.nio.ByteBuffer
+import java.text.SimpleDateFormat
+import java.util.zip.CRC32
+
+@Component
+open class MessageUtils {
```

Comment:
I can't see the unit tests for this class. HTME has a comprehensive set which can easily be imported and run as part of the circleci build step.

Reply:
Seconded.